modules/claims_api/app/workers/claims_api/claim_establisher.rb

```diff
@@ -20,11 +21,21 @@ def perform(auto_claim_id) # rubocop:disable Metrics/MethodLength
       form_data = auto_claim.to_internal
       auth_headers = auto_claim.auth_headers
 
-      response = service(auth_headers).submit_form526(form_data)
-      ClaimsApi::Logger.log('526',
-                            claim_id: auto_claim_id,
-                            vbms_id: response.claim_id)
-      auto_claim.evss_id = response.claim_id
+      if Flipper.enabled? :claims_status_v1_lh_auto_establish_claim_enabled
+        response = service(auth_headers).submit(auto_claim, form_data)
+
+        ClaimsApi::Logger.log('526', claim_id: auto_claim_id,
+                                     vbms_id: response[:claimId])
+
+        auto_claim.evss_id = response[:claimId]
+      else
+        response = service(auth_headers).submit_form526(form_data)
+        ClaimsApi::Logger.log('526', claim_id: auto_claim_id,
+                                     vbms_id: response.claim_id)
+
+        auto_claim.evss_id = response.claim_id
+      end
```

Comment:
if we were doing the calls alone I would say we could just duplicate this and provide with indifferent access to access both responses... but at the same time, I kind of like the idea of keeping these separate under the change of modifying the logger so we _know_ which service this came from. Let's modify the `'526'` in the logger to be something like `'526_docker_container'` so we can distinguish that we had a success with the docker container. Then we'll know if it failed calling the docker container specifically. Sure we can do this by knowing the flipper feature is on, but I'd prefer to distinguish the two.